tasks/ami.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import boto3
```

Comment:
I think that this should be used in local scope, otherwise it will create issues in many images (I don't think that it is installed everywhere).

Maybe you could make a function `get_ec2` that relies on that?

Reply:
I'll go with a local import!